.github/workflows/veracode-scans.yml

```diff
@@ -0,0 +1,126 @@
+# Veracode pipeline scans
+on:
+  workflow_call:
+    inputs:
+      module_directory:
+        required: true
+        type: string
+      module_name:
+        required: true
+        type: string
+      major_version:
+        required: true
+        type: string
+      minor_version:
+        required: true
+        type: string
+      target_branch_name:
+        required: true
+        type: string
+      event_name:
+        required: true
+        type: string
+      threshold_pipeline:
+        required: true
+        type: string
+      threshold_sandbox:
+        required: true
+        type: string
+      cache_key:
+        required: true
+        type: string
```

Comment:
since the cache is used internally in the workflow, generate inside it
```suggestion
```

Reply:
and update the cache key to `${{ github.sha }}-${{ github.run_id }}-veracode-binary`